### PR TITLE
[MM-60832] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx' with FormattedMessage from “react-intl”

### PR DESCRIPTION
**Pull request by Niharika0104**
#### Summary
Replace FormattedMarkdownMessage in 'webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx' with FormattedMessage from “react-intl”

#### Ticket Link

Fixes: [#28602](https://github.com/mattermost/mattermost/issues/28602)
JIRA: [MM-60832](https://mattermost.atlassian.net/browse/MM-60832)

#### Release Note

```release-note
NONE
```

[MM-60832]: https://mattermost.atlassian.net/browse/MM-60832?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx [hunk 1]
```diff
@@ -200,8 +202,18 @@ export default class CustomTermsOfServiceSettings extends AdminSettings<Props, S
                 <BooleanSetting
                     key={'customTermsOfServiceEnabled'}
                     id={'SupportSettings.CustomTermsOfServiceEnabled'}
-                    label={<FormattedMessage {...messages.enableTermsOfServiceTitle}/>}
-                    helpText={<FormattedMarkdownMessage {...messages.enableTermsOfServiceHelp}/>}
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx:204]: only this has the internal link, so this must be changed others can be kept as is. See this PR for reference https://github.com/mattermost/mattermost/pull/28635

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx (hunk 1 above)
Comment (Niharika0104) [reply webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx:204]: @M-ZubairAhmed Currently I don't see any internal link here or perhaps taking reference from the #28635 do you want me to include this in the values 
```
a: (chunks: string) => (
                                <Link to='/admin_console/environment/file_storage'>
                                    {chunks}
                                </Link>
                            ) 
```
for every FormattedMessage tag I used ` <b></b>` tags in the values are you fine with this or should i remove them as well?

Comment (M-ZubairAhmed) [reply webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx:204]: you would have to check if each message if it contains markdown or not. If it does then you should apply suitable HTML replacement like for bold -> `<b>`

Comment (Niharika0104) [reply webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx:204]: Okay ..I will check that.
What about the link that you were mentioning ,I don't see any internal link that you were talking about?

Comment (M-ZubairAhmed) [reply webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx:204]: i meant that in `enableTermsOfServiceHelp` i dont any bold markdown but you have added that! i can see only link being used there. thats why i asked you to recheck all https://github.com/mattermost/mattermost/pull/28690#discussion_r1798669476

**Comment by M-ZubairAhmed** [comment]
@Niharika0104 do you need help in progressing this one?

**Comment by Niharika0104** [comment]
@M-ZubairAhmed Can you check it now? I made all the changes.

File: webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx [hunk 2]
```diff
@@ -200,8 +199,13 @@ export default class CustomTermsOfServiceSettings extends AdminSettings<Props, S
...
+                    label={<FormattedMessage {...messages.enableTermsOfServiceTitle} />}
+                    helpText={<FormattedMessage
+                        {...messages.enableTermsOfServiceHelp}
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx:204]: `enableTermsOfServiceHelp` string has not been updated with the XML `<a/>`

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by M-ZubairAhmed** [comment]
@Niharika0104 there are lots of lint errors as well please fix those

File: webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx (hunk 2 above)
Comment (Niharika0104) [reply webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx]: @M-ZubairAhmed Can you check it now?

Comment (M-ZubairAhmed) [reply webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx]: thanks for adding the translation also need to be edited in en.json as well

Comment (Niharika0104) [reply webapp/channels/src/components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.tsx]: @M-ZubairAhmed I updated it

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Comment by varghesejose2020** [comment]
@lindy65  please assign me i will test this PR

**Comment by lindy65** [comment]
Assigned to you for testing @varghesejose2020 👍

**Comment by varghesejose2020** [comment]
@lindy65. The changes Tested and it working as expected LGTM🚀

**Comment by lindy65** [comment]
Thanks for testing @varghesejose2020 - counted towards Hacktoberfest

**Pull request merged**